_posts/en/newsletters/2021-02-10-newsletter.md

```diff
@@ -0,0 +1,231 @@
+     that are smaller in size but sufficient to uniquely identify
+     transactions. A node receiving a compact block matches each short id with a
+     transaction in its mempool to reconstruct the full block. This greatly
+     reduces block relay bandwidth."
+
+  a2link="https://bitcoincore.reviews/20799#l-90"
+
+  q3="What is the difference between \"high bandwidth\" and \"low bandwidth\"
+     modes?"
+
+  a3="In high bandwidth mode, compact blocks are sent unsolicited, trading
+     higher bandwidth use for improved latency, whereas in low bandwidth mode
+     they are sent only on request after an `inv` or `headers` message was
+     received. In high bandwidth mode, compact block messages can be relayed
+     prior to full validation, requiring only that the block header be valid
+     before relay."
+
+  a3link="https://bitcoincore.reviews/20799#l-156"
+
+  q4="How do we choose which peers are in high bandwidth mode?"
+  a4="We select [up to three peers][three] that recently sent us a novel valid
+     block. This logic is in the net processing function,
+     `MaybeSetPeerAsAnnouncingHeaderAndIDs`."
+  a4link="https://bitcoincore.reviews/20799#l-219"
+
+  q5="Why can we stop supporting version 1 compact blocks?"
+
+  a5="BIP152 supports two versions: version 1 (without witnesses) and version 2
+     (with witnesses). Version 2 is required to reconstruct segwit
+     blocks. Segwit was activated in August 2017, so providing version 1
+     pre-segwit compact blocks to peers is no longer useful; without the
+     witnesses, peers are unable to validate the consensus rules on
+     them."
+
+  a5link="https://bitcoincore.reviews/20799#l-104"
+
+  q6="Practically speaking, how can version 1 support be dropped?"
+  a6="By ignoring incoming `sendcmpct` messages with version equal to 1, no
+     longer sending `sendcmpct` messages that signal version 1 support, only
+     sending `sendcmpct` messages to `NODE_WITNESS` peers, and responding to
+     `sendcmpct` and `blocktxn` messages with witness-serialized blocks and
+     transactions."
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [HWI #433][] adds support for signing PSBTs with OP_RETURN outputs.
+
+- [Bitcoin Core #19509][] adds per-peer message capture between nodes as well
+  as the ability to produce JSON outputs from those logs.  Using the newly
```

Comment:
I like it better the way it is (though I'd probably not have pluralized "outputs"; if that's your concern, we could possibly just drop that word without otherwise changing the sentence).

Reply:
I wanted to distinguish between logs (which I'd generally understand as a file that records specific events in the code) and what we have here, which is message dumps/message capture. It's not very important.